testsuite/tests/kuadrant/authorino/dinosaur/conftest.py

```diff
@@ -28,6 +29,18 @@ def admin_rhsso(blame, keycloak):
     return info
 
 
+@pytest.fixture(scope="module", autouse=True)
+def commit(request, authorization):
+    """Commits all important stuff before tests"""
+    request.addfinalizer(authorization.delete)
+    try:
+        authorization.commit()
+        authorization.wait_for_ready()
+    except OpenShiftPythonException as exc:
+        if "Too many" in exc.result.err():
+            pytest.xfail("AuthPolicy max limit")
```

Comment:
This xfail usage is basically a skip when used as a function. Therefore when the product get fixed we need to watch for the lack of xfails for the dinosaur tests in testrun log to notice the issue was fixed.

Reply:
when authorino gets fixed, this wont trigger and will pass